src/__tests__/lib/command/util/organizations-util.test.ts

```diff
@@ -2,8 +2,17 @@ import { jest } from '@jest/globals'
 
 import type {
 	OrganizationResponse,
+	OrganizationsEndpoint,
+	SmartThingsClient,
 } from '@smartthings/core-sdk'
 
+import {
+	ValueTableFieldDefinition,
```

Comment:
Does this want to be identified as a `type`?

Reply:
Yup, fixed.